lib/plugins/features/movies/controller.js

```diff
@@ -6,5 +6,41 @@ exports.create = (payload) => {
 
   return new Movie().save(payload)
   .then((movie) => new Movie({ id: movie.id }).fetch());
-  
+
 };
+
+exports.display = (params) => {
+
+  if (params === undefined || Object.keys(params).length === 0) {
```

Comment:
Talked IRL: This code can be made DRYer by turning the conditionals inside out. Each one creates a movie model, and each one fetches all. The only difference is in how the query is built. Thus, by flipping it inside out, it can become:

```js
return new Movie()
.query((qb) => {

  ...

  if(<something>) {
    qb.where(<something>);
  }  

  ...

})
.fetchAll();
```